Web/Scotty/Action.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE CPP               #-}
+{-# LANGUAGE NamedFieldPuns    #-}
```

Comment:
What is this for?

Reply:
This allows to read record fields in most natural way (in my taste):
```hs
someExceptionHandler Options{verbose} =
    when (verbose > 0) ...
```